momento-sdk/src/intTest/java/momento/sdk/ListTest.java

```diff
@@ -231,4 +269,122 @@ public void shouldFetchAllValuesWhenListFetchWithInvalidIndices() {
 
     assertThat(cacheListFetchResponse).isInstanceOf(CacheListFetchResponse.Error.class);
   }
+
+  @Test
+  public void listConcatenateFrontStringHappyPath() {
+    List<String> oldValues = Arrays.asList("val1", "val2", "val3");
```

Comment:
You don't need to change this now, but we should make sure the same value can appear twice in a list.